style.css

```diff
@@ -0,0 +1,214 @@
+@import url('https://fonts.googleapis.com/css2?family=Nabla&family=Poppins&display=swap');
+
+/* minimum mobile size 375px */
+
+@media screen and (min-width: 375px) {
+  body {
+    margin: 0;
+    padding: 0;
+  }
+
+  #header {
+    width: 100%;
+    height: 70vh;
+  }
+
+  .head {
+    display: flex;
+    width: 100%;
+    justify-content: space-between;
+    align-items: center;
+    position: relative;
+  }
+
+  #logo {
+    height: 20px;
+    font-family: 'nabla', cursive;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 200%;
+    line-height: 20px;
+    padding: 0% 5%;
+  }
+
+  .menu-icon {
+    width: 20%;
+  }
+
+  /* section 2 */
+
+  .presentation {
+    background-image: url("headerBackground.svg");
+    background-repeat: no-repeat;
+    background-size: cover;
+    height: 85vh;
+    display: flex;
+    width: 100%;
+    flex-direction: column;
+  }
+
+  .greeting {
+    width: 45%;
+    height: 200px;
+    font-family: 'poppins', sans-serif;
+    font-style: normal;
+    font-weight: 800;
+    font-size: 1.8em;
+    line-height: 60px;
+    display: block;
+    margin: 0% 5%;
+    margin-top: 20%;
+    align-items: center;
+    color: #172b4d;
+    position: relative;
+  }
+
+  .self-description {
+    width: 75%;
+    height: 250px;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 1.2em;
+    line-height: 140%;
+    color: #344563;
+    margin-left: 5%;
+    position: relative;
+    margin-bottom: 10px;
+  }
+
+  .media-connect {
+    display: flex;
+    flex-direction: column;
+    margin: 3% 5%;
+    margin-top: 0;
+  }
+
+  .media-connect p {
+    width: 60%;
+    font-family: 'sans-serif', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 1.3em;
+    line-height: 24px;
+    color: #7f8cff;
+    position: relative;
+    margin-top: 0%;
+  }
```

Comment:
- [ ] Kindly use the `Poppins` don't for this element. That way, your design will look identical to that of the Figma template that you chose. 👍  

#### Hint: You can declare the Poppins family at the beginning of the stylesheet since that is the main font used for this template on Figma. For you to use it, you can do this:

```CSS
* {
  font-family: 'Poppins', 'sans-serif';
}
```